erts/emulator/beam/erl_msacc.h

```diff
@@ -431,9 +432,10 @@ void erts_msacc_set_bif_state(ErtsMsAcc *msacc, Eterm mod, void *addr);
 #define ERTS_MSACC_SET_STATE_CACHED_M_X(state) ERTS_MSACC_SET_STATE_CACHED_M(state)
 #define ERTS_MSACC_POP_STATE_M_X() ERTS_MSACC_POP_STATE_M()
 #define ERTS_MSACC_PUSH_AND_SET_STATE_M_X(state) ERTS_MSACC_PUSH_AND_SET_STATE_M(state)
-#define ERTS_MSACC_SET_BIF_STATE_CACHED_X(Mod,Addr)       \
-    if (ERTS_MSACC_IS_ENABLED_CACHED_X())               \
-        erts_msacc_set_bif_state(__erts_msacc_cache, Mod, Addr)
+#define ERTS_MSACC_SET_BIF_STATE_CACHED_X(Mod,Addr)                       \
+    if (ERTS_MSACC_IS_ENABLED_CACHED_X()) {                               \
+        (void)erts_msacc_set_bif_state(__erts_msacc_cache, Mod, Addr)     \
+    }
```

Comment:
```suggestion
#define ERTS_MSACC_SET_BIF_STATE_CACHED_X(Mod,Addr) do {                  \
    if (ERTS_MSACC_IS_ENABLED_CACHED_X()) {                               \
        (void)erts_msacc_set_bif_state(__erts_msacc_cache, Mod, Addr);    \
    } \
    } while(0)
```

Nightly tests are failing with msacc extra. I think the above fix is correct.

Reply:
Woops! :D

Thanks!